app/js/main.js

```diff
@@ -658,7 +675,31 @@ function _getDefaultDirectory() {
 }
 
 /**
-* COnverting frames to png
+ * Check the save directory still exists and recreate it if deleted.
```

Comment:
Reword to:
"Create the default save directory if needed."

Simple and to the point.